x/model/keeper/msg_server_model_version.go

```diff
@@ -44,6 +45,11 @@ func (k msgServer) CreateModelVersion(goCtx context.Context, msg *types.MsgCreat
 		return nil, types.NewErrModelVersionAlreadyExists(msg.Vid, msg.Pid, msg.SoftwareVersion)
 	}
 
+	_, err = base64.StdEncoding.DecodeString(msg.OtaChecksum)
+	if err != nil {
+		return nil, types.NewErrOtaChecksumIsNotValid(msg.OtaChecksum)
+	}
```

Comment:
Can we move this validation to the 'ValidateBasic()' method?

We also need to validate it when updating the model.